logical_router_static_route.go

```diff
@@ -185,8 +184,13 @@ func (odbi *ovndb) lrsrListImp(lr string) ([]*LogicalRouterStaticRoute, error) {
 					return nil, fmt.Errorf("Unsupport type found in ovsdb rows")
 				}
 			}
+			lrFound = true
 			break
 		}
 	}
+
+	if !lrFound {
+		return nil, ErrorNotFound
+	}
```

Comment:
why not simply check 
if len(listLRSR) == 0 {
  return nil, ErrorNotFound
}

Reply:
Because it is ok to not have any lsrs for a given router.

Reply:
agree. this is better ;)